doc/content/documentation/functions-and-recursion.md

```diff
@@ -99,11 +99,11 @@ Calls the function with the given arguments. The last argument must be a list of
 
 ## Passing by reference
 
-Sometimes it is required that a variable should pass to a function by reference. This can be done by applied the `:reference` metadata to the symbol.
+Sometimes it is required that a variable should pass to a function by reference. This can be done by applying the `:reference` metadata to the symbol.
 
 ```phel
 (fn [^:reference my-arr]
   (php/apush my-arr 10))
 ```
 
-Support for references is very limited in Phel. Currently, it only works for function arguments (except destructuring).
\ No newline at end of file
+Support for references is very limited in Phel. Currently, it only works for function arguments (except destructuring).
```

Comment:
What's the change in this line? 😆

Reply:
Newline is missing. But that no problem. I will merge it!

Reply:
Yes, it make sense. I thought it might be that 🙈👌🏻